### DISCUSSION TITLE
Does vips support chinese characters when converting from pdf to jpeg

### DISCUSSION THREAD

**Discussion by revathi-murali** [libvips/libvips#2598] [Q&A]
Hi,

When a pdf with chinese characters are converted to jpeg using vips, the output jpeg does not contain those chinese characters and they are blank. 

**Note**: Since its a customer file, I wont be able to share it.

Vips verion: 8.11.4
Pdfium: 4666

Any special setting that we need to do?

**Comment by jcupitt** [comment]
Hi again, it should work, I think. Perhaps it's a PDF without an embedded font? In which case you'll need to add the font to your system before conversion.

**Reply by revathi-murali** [reply]
When I ran `pdffonts` command on the pdf, I see the following output

<img width="798" alt="Screenshot 2021-12-24 at 7 56 21 PM" src="https://user-images.githubusercontent.com/2732287/147359083-e2075ce5-0f98-4701-9f0c-5959695d9781.png">


Looks like those fonts are not embedded. So then, do I need to install them manually?
